cronet/src/main/java/io/grpc/cronet/CronetClientTransport.java

```diff
@@ -166,13 +166,10 @@ public Runnable start(Listener listener) {
     synchronized (lock) {
       started = true;
     }
-    return new Runnable() {
-      @Override
-      public void run() {
-        attrs = CronetClientTransport.this.listener.filterTransport(attrs);
-        // Listener callbacks should not be called simultaneously
-        CronetClientTransport.this.listener.transportReady();
-      }
+    return () -> {
```

Comment:
We can merge this change, but I do think we want to avoid going around replacing code with lambdas just because we can. The anonymous class is still perfectly acceptable.

(It's much more understandable with the `Runnable::run` in the test.)

Reply:
Makes Sense, Updated